paraatm/bdn/_bdn.py

```diff
@@ -0,0 +1,157 @@
+"""
+Bayesian deep neural network (BDN) class
+---
+Required pakages:
+keras
+numpy
```

Comment:
From what I can tell, `tensorflow` is also required.  First I installed `keras` only, and I got this error:
```
ModuleNotFoundError: No module named 'tensorflow'
```

Reply:
Yes you're right. Tensorflow is a backend of keras. I will modify the required packages later.